GameLogic/Engine/World.py

```diff
@@ -107,6 +107,10 @@ def y(self) -> int:
 	def z(self) -> int:
 		return self._impl.z
 
+	@property
+	def position(self) -> Vector2D:
```

Comment:
It's equal to pos.xy(). I don't think that we need excess field.

It would be better add Item.ClickTile(self, tile) method, instead of pass tile as an object to Item.InteractWith(self, object) 🙃